kinetic/distribution.yaml

```diff
@@ -5713,6 +5713,15 @@ repositories:
       type: git
       url: https://github.com/ros-geographic-info/open_street_map.git
       version: master
+    release:
+      packages:
+      - osm_cartography
+      - route_network
+      - test_osm
```

Comment:
Just checking: do you want this test package to be released for users to download?